client/asset/eth/eth.go

```diff
@@ -773,22 +781,28 @@ func (eth *ExchangeWallet) Redeem(form *asset.RedeemForm) ([]dex.Bytes, asset.Co
 		}
 		redeemedValue += swapData.Value
 		unlocked += redemption.UnlockedReserves
-		inputs = append(inputs, redemption.Spends.Coin.ID())
+		//inputs = append(inputs, redemption.Spends.Coin.ID())
 	}
 
 	outputCoin := eth.createFundingCoin(redeemedValue)
 	fundsRequired := dexeth.RedeemGas(len(form.Redemptions), contractVersion) * form.FeeSuggestion
 
 	// TODO: make sure the amount we locked for redemption is enough to cover the gas
 	// fees. Also unlock coins.
-	_, err := eth.node.redeem(eth.ctx, form.Redemptions, form.FeeSuggestion, contractVersion)
+	tx, err := eth.node.redeem(eth.ctx, form.Redemptions, 200 /*form.FeeSuggestion*/, contractVersion)
 	if err != nil {
 		return fail(fmt.Errorf("Redeem: redeem error: %w", err))
 	}
 
 	eth.UnlockReserves(unlocked)
 
-	return inputs, outputCoin, fundsRequired, nil
+	txHash := tx.Hash()
+	txs := make([]dex.Bytes, len(form.Redemptions))
+	for i, _ := range txs {
+		txs[i] = txHash[:]
+	}
+
+	return txs /*inputs*/, outputCoin, fundsRequired, nil
```

Comment:
The server redeem coin wants the redeeming tx. Does this mess up balance calculations on the server side?

Reply:
Changed it to the tx hashes.